src/config.c

```diff
@@ -447,6 +447,8 @@ void loadServerConfigFromString(char *config) {
 			}
 			server.pm_file_size = size;
 			server.use_volatile = true;
+        } else if (!strcasecmp(argv[0],"keys-on-pm") && (argc == 2)) {
+            server.keys_on_pm = (!strcasecmp(argv[1], "on") ? true : false);
```

Comment:
consider validation of this parameter - now every value different than "on" is treated as "off".
If You look to other parameters it validate parameter properly value range, like for example methods: yesnotoi  or configEnumGetValue